src/main/play-doc/developer/DevQuickStart.md

```diff
@@ -51,12 +51,12 @@ object Global extends GlobalSettings {
 1. Add `sbt-conductr` to the `project/plugins.sbt`:
 
     ```scala
-    addSbtPlugin("com.typesafe.conductr" % "sbt-conductr" % "1.0.0")
+    addSbtPlugin("com.typesafe.conductr" % "sbt-conductr" % "1.0.1")
     ```
-2. Enable the plugin in the `build.sbt`:  
+2. Enable the plugin in the `build.sbt` (again assuming a Play 2.4 Scala application):  
```

Comment:
I would skip this step. In the quick start we assume they are using a Play application and a Play application has already enabled `PlayScala` or `PlayJava`. I like the more detailed documentation in `DeployingBundles.md` though.